content/docs/getting-started/deploy-app/deploy-cli.md

```diff
@@ -14,4 +14,147 @@ seo:
   description: "" # custom description (recommended)
   canonical: "" # custom canonical URL (optional)
   noindex: false # false (default) or true
+asciinema: true
 ---
+
+Besides using the API directly or clicking through the dashboard UI, you can also interact with Oakestra using the Oakestra Command Line Interface.
+
+{{< callout context="caution" title="Requirements" icon="outline/alert-triangle">}}
+  - You have a running Oakestra setup.
+  - You can access the APIs at `<IP_OF_CLUSTER_ORCHESTRATOR>:10000`
+  - You have read and familiarized yourself with the [API]({{< relref "deploy-api.md" >}}) and [Dashboard]({{< relref "deploy-dashboard.md" >}}) approaches.
```

Comment:
Is this really a requirement? I think that I can use the CLI even if I never used the API and the Dashboard right?

Reply:
Ok I have removed that line for now - My idea is that it is very beneficial to know about the underlying things the CLI uses and to know about the alternatives - and yes I agree this is not a hard requirement